python/src/main/python/drivers/glsl-to-spv-worker.py

```diff
@@ -209,51 +217,54 @@ def doImageJob(args, imageJob):
 
 ################################################################################
 
+
 def get_service(server, args, worker_info_json_string):
     try:
-        httpClient = THttpClient.THttpClient(server)
-        transport = TTransport.TBufferedTransport(httpClient)
+        http_client = THttpClient.THttpClient(server)
+        transport = TTransport.TBufferedTransport(http_client)
         protocol = TBinaryProtocol.TBinaryProtocol(transport)
         service = FuzzerService.Client(protocol)
         transport.open()
 
         # Get worker name
-        platforminfo = worker_info_json_string
+        platform_info = worker_info_json_string
 
-        tryWorker = args.worker
-        print("Call getWorkername()")
-        workerRes = service.getWorkerName(platforminfo, tryWorker)
-        assert type(workerRes) != None
+        try_worker = args.worker
+        print("Call getWorkerName()")
+        worker_res = service.getWorkerName(platform_info, try_worker)
+        assert type(worker_res) is not None
 
-        if workerRes.workerName == None:
-            print('Worker error: ' + tt.WorkerNameError._VALUES_TO_NAMES[workerRes.error])
+        if worker_res.workerName is None:
+            # noinspection PyProtectedMember
+            print('Worker error: ' + tt.WorkerNameError._VALUES_TO_NAMES[worker_res.error])
             exit(1)
 
-        worker = workerRes.workerName
+        worker = worker_res.workerName
 
         print("Got worker: " + worker)
         assert(worker == args.worker)
 
-        if not os.path.exists(args.worker):
-            os.makedirs(args.worker)
+        os.makedirs(args.worker, exist_ok=True)
 
         # Set working dir
         os.chdir(args.worker)
 
         return service, worker
 
-    except (TApplicationException, ConnectionRefusedError, ConnectionResetError) as exception:
+    except (TApplicationException, ConnectionRefusedError, ConnectionResetError):
         return None, None
 
 ################################################################################
 
-def isDeviceAvailable(serial):
+
+def is_device_available(serial):
     cmd = 'adb devices'
-    devices = subprocess.run(cmd, shell=True, universal_newlines=True, stdout=subprocess.PIPE, timeout=2).stdout.splitlines()
+    devices = subprocess.run(cmd, shell=True, universal_newlines=True, stdout=subprocess.PIPE,
+                             timeout=2).stdout.splitlines()
```

Comment:
Should we use a larger timeout here?  If the timeout is exceeded the worker quits, so timing out is kind of a last resort.  So maybe use 5?  Also could we use a named constant for this purpose?